src/common/Error/Index.js

```diff
@@ -0,0 +1,28 @@
+import { StyledNavLink } from "../Header/Navigation/styled";
+import { Wrapper, ErrorIcon, Title, Subtitle, Button } from "./styled";
+import { MainWrapper } from "../MainWrapper";
+
+export const Error = () => {
```

Comment:
I think we should export this Component as `default`, for example, the Loader component was exported as `default`, so we should stick to this convention.